core/systems/performance/performance_manager.gd

```diff
@@ -29,30 +31,22 @@ var library_item: LibraryLaunchItem
 var platform_provider: PlatformProvider
 var profile: PerformanceProfile
 var profile_state: String # docked or undocked
```

Comment:
We should use an enum for this state to more strongly type what states it can be in.

Reply:
I suppose. We'll need to convert it to string so the profile names make sense.